### PR TITLE
Feature/Aggregate-tables-able-to-plot-time-series-data- Part 1

### PR DESCRIPTION
**Pull request by nobert-mumo**
We want users to be able to group data by time periods e.g month, year,quarter, etc on the aggregate table. This PR introduced a column `AsOfDate` which is the end of month for each month. This will enable time series analysis on the aggregates table

Jira ticket: https://thepalladiumgroup.atlassian.net/browse/KHP3-3606

### CODE REVIEW

**Review by Marymary-dev — APPROVED** [review]

File: Scripts/REPORTING/load_aggregate_hts_months_last_test.sql
Comment (cbrianbet) [inline Scripts/REPORTING/load_aggregate_hts_months_last_test.sql:1]: Hi @nobert-mumo I see you are changing the table for this dataset from AggregateHTSMonthsLastTest to AggregateHTSPNSSexualPartner

**Review by cbrianbet — CHANGES_REQUESTED** [review]

File: Scripts/REPORTING/load_aggregate_hts_months_last_test.sql
Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_hts_months_last_test.sql:1]: @cbrianbet good catch! oops. will correct that.

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_hts_months_last_test.sql:1]: @cbrianbet made the correction.

**Comment by cbrianbet** [comment]
👍

**Pull request merged**